.changeset/forty-rocks-jog.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-kubernetes': minor
```

Comment:
Since this package has [major version zero](https://semver.org/#spec-item-4) (v0.9.3 at the time of writing) and this change is backwards-compatible
```suggestion
'@backstage/plugin-kubernetes': patch
```